Button and Contributor styling

lortza opened the pull request:
Resources page buttons: turned the edit and trash icons into buttons and floated them to the right to make them more consistent looking and easier to click.

About page: added styling and images to the project contributors

siakaramalegos commented on db/seeds.rb:
```diff
@@ -5,3 +5,51 @@
+    # t.string   "name"
+    # t.text     "description"
+
+Category.create!([
```
So, there is already a `rake db:populate` task that populates fake category and resource data.  Also, production is already live so not sure the usefulness of a seed file.  Everything else sounds great though of course I haven't seen it yet.  :)  If you want some of these categories in production, just make them on the site.

siakaramalegos commented:
Hey girl - if you just take out the seeds change, I can pull everything else in.

lortza commented:
@siakaramalegos, I just removed the seeds, so you should be good to go. :)

siakaramalegos commented:
Love the new contributors page!  Great job.  Deploying now so hopefully will be up in a sec. I added a 10px margin to the resource-icons so they link up better with the resource titles.  I also fixed some of the column responsiveness on the categories page.

lortza commented:
Nice! I'm glad I could finally contribute.

Anne Richardson
portfolio http://localflavormarketing.com/ | linkedin
https://www.linkedin.com/in/localflavor | github
https://github.com/lortza | side projects
https://www.thegamecrafter.com/designers/anne-richardson | travel blahg
http://viajaryamar.com/

On Wed, Nov 11, 2015 at 2:55 PM, Sia notifications@github.com wrote:

> Love the new contributors page! Great job. Deploying now so hopefully will
> be up in a sec. I added a 10px margin to the resource-icons so they link up
> better with the resource titles. I also fixed some of the column
> responsiveness on the categories page.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/siakaramalegos/tts_resources/pull/27#issuecomment-155907132
> .
